exec/Chainweb/Backfill.hs

```diff
@@ -27,7 +28,9 @@ backfill e@(Env _ c _ _) = withPool c $ \pool -> do
   putStrLn "Backfilling..."
   cont <- newIORef 0
   mins <- minHeights pool
-  traverseConcurrently_ Par' (f pool cont) $ lookupPlan mins
+  let !count = M.size mins
+  printf "Valid Chains: %d\n" count
+  when (count == 10) $ traverseConcurrently_ Par' (f pool cont) $ lookupPlan mins
```

Comment:
We probably shouldn't have the constant `10` here.  It should get it from the node.